### DISCUSSION TITLE
[Feature]: To use Fluent UI 8 based components with Fluent UI 9

### DISCUSSION THREAD

**Discussion by kroudams** [microsoft/fluentui#25671] [Q&A] [answered]
### Library

React Components / v9 (@fluentui/react-components)

### Describe the feature that you would like added

Is it possible to use Fluent UI 8 based components with Fluent UI 9?

For example, if I already have some utility components (e.g. Input, EmptyState, etc...) and I want to use them in my Fluent UI 9 based component implementation.

### Validations

- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by GeoffCoxMSFT** [comment, marked as answer]
@kroudams - v9 was design to be incrementally adopted and work side-by-side with v8 components.  Check out the migration topics at https://react.fluentui.dev for more information.

**Comment by ling1726** [comment]
Converting this to a discussion, since this is more about an how to do something with Fluent. Happy to turn this into an issue again if there is any action needed on our end
